static/app/views/settings/account/apiApplications/details.tsx

```diff
@@ -105,7 +105,7 @@ class ApiApplicationsDetails extends DeprecatedAsyncView<Props, State> {
                       <StyledSpan>
                         <em>hidden</em>
                       </StyledSpan>
-                      <StyledButton onClick={this.rotateClientSecret}>
+                      <StyledButton onClick={this.rotateClientSecret} priority="danger">
```

Comment:
hows danger look? mind adding a screenshot?

Reply:
Let's also make this button a bit bigger to match the size of the form inputs?